package.json

```diff
@@ -55,7 +55,7 @@
     "@skyux/config": "4.4.0",
     "@skyux/core": "4.5.0",
     "@skyux/docs-tools": "4.9.0",
-    "@skyux/forms": "4.17.0",
+    "@skyux/forms": "4.17.1",
```

Comment:
Don't we need to update the `popovers` version in line 3 of this file?

Reply:
Oops, didn't commit all my files. Good catch.